src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -305,6 +305,16 @@ uint64_t ImageReplayer<I>::get_remote_snap_id_end_limit() {
   return CEPH_NOSNAP;
 }
 
+template <typename I>
+uint64_t ImageReplayer<I>::get_last_snapshot_bytes() const {
```

Comment:
Is there a better way to do this? The ImageReplayer is mode agnostic for the most case. This would at the vert least need to check that it is not journal mode before returning.